sdks/typescript/package.json

```diff
@@ -32,6 +32,7 @@
     "chai": "^4.3.4",
     "date-fns": "^2.28.0",
     "fast-deep-equal": "^3.1.3",
+    "find-git-root": "^1.0.4",
```

Comment:
A few suggestions that don't need action:
- It is usually advisable to avoid using modules that are not actively maintained and/or do something trivial because of the eventual [dependency bloat](https://news.ycombinator.com/item?id=22830284). It also makes identifying what packages are dev dependencies v/s ones that are actually needed at runtime hard amongst a host of other issues. 
- Eventually, we should [evaluate the strictness](https://blog.softwaremill.com/it-depends-the-art-of-dependency-management-in-javascript-f1f9c3cde3f7) of our [dependency versioning](https://nodejs.dev/learn/semantic-versioning-using-npm). Having a `package-lock.json` file solves the problem that arises from dependency upgrades only if the upgrades are within our bounds of expectations.

Reply:
Ah, I though the trend in node was to have lots of tiny packages to share code (kind of like the unix philosophy for binaries) rather than re-invent everything. But maybe that's a statement of how things are, not how they should be :). 

Yeah, we should re-consider strictness too.

Reply:
You're right in the principle of having shared tiny packages. Problem (as evidenced over the last decade) is that the scope of what is considered to be a module is arbitrarily small. Loads of troll modules exist that do nothing comprehensive and eventually add to operational toil (versions, lock files, security checks etc.).

I usually see the code of the module and determine if it doing "reasonable" work to add it as a dependency. There could also be a trade-off some times since enthusiastic module authors lean strongly towards "generalize everything" (as they should) which may not serve long term evolving needs. This usually leads me to adopt the generalized code and tweak it to our specific use case instead of depending on a generalized module.